frontend/src/components/bs/components/spaces-table/helpers/make-link.helper.tsx

```diff
@@ -0,0 +1,16 @@
+import { Link } from 'components/common/common';
+import { AppRoute } from 'common/enums/app/app-route.enum';
+import styles from '../../../styles.module.scss';
+
+const makeLink = (name: string, id: string): JSX.Element => {
+  return (
+    <Link
+      className={styles.link}
+      to={`${AppRoute.BS_SPACE}/${id}` as unknown as AppRoute.BS_SPACE_$ID}
+    >
+      {name}
+    </Link>
+  );
+};
```

Comment:
I do not see any reason for creating this helper. 🤔  What are you trying to reuse here? But If you need to, then you can create component `BsSpaceLink`

Reply:
In the Get rows helper I can pass or name or JSX component, I need to give the user the ability to open the space page, so I made it in a way of clicking on the name of the space.

Reply:
Just move helper to components folder of the bs component(page with a table of spaces) and rename it?)

Reply:
Yep. And make it a functional component instead of just function

Reply:
Yep, agree, it is component)